### DISCUSSION TITLE
Session won't start?

### DISCUSSION THREAD

**Discussion by joseamonteironeto** [devlikeapro/waha#398] [Q&A]
Hi! I can't generate a QR CODE because instance stays STARTING forever, tried on different devices, and it repeats the same behaviour... Am i'm doing something wrong? Or is this a bug?

**Comment by devlikepro** [comment]
Hi! What version do you use? Make sure you're using latest `devlikeapro/waha` docker image

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
- https://boosty.to/wa-http-api/posts/a6c01a2d-704e-4066-978a-890e84de9da1
- https://boosty.to/wa-http-api/posts/6a58df1a-6e47-4ed2-a2b0-b673e99f2070

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
